pixi_diff_to_markdown/cli.py

```diff
@@ -44,9 +45,13 @@ def main(
             help="Whether to hide tables in a collapsible element.", show_default=False
         ),
     ] = None,
-):
+) -> int:
```

Comment:
```suggestion
):
```
i don't think this does anything